src/universal/redux/reducers/currentTemplate/actionCreators.js

```diff
@@ -29,3 +29,14 @@ export function promoteTemplateToProductionVersion(projectId, templateId, versio
     }
   };
 }
+
+export function loadTemplateVersionHistory(versionId) {
```

Comment:
It loads an older `version` from `versions` list. I think if you don't pass an existing `versionId` it will load the current one, yes. The logic is in the `reducer`, not here.